### PR TITLE
AbuenCLI-TemplateAuth (HOLD_TO_MERGE)

### PR DESCRIPTION
**Pull request by Abuen**
# Type of change? 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: solutions/remotemonitoring/armtemplates/standard.json [hunk 1]
```diff
@@ -855,7 +868,98 @@
...
             "properties": {}
-        }
+        },
+        {
+            "properties": {
+              "state": "Enabled",
+              "definition": {
+                "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+                "actions": {
+                  "Send_an_email": {
```
[the remark this answers is not in the document]
Comment (Abuen) [reply solutions/remotemonitoring/armtemplates/standard.json]: Send_an_email is the name of the logic app connector we want to use. changing the name would change the type of connector

File: solutions/remotemonitoring/armtemplates/standard.json [hunk 2]
```diff
@@ -855,7 +868,98 @@
...
+                            "type": "string"
+                          }
+                        },
+                        "type": "object"
+                      }
+                    }
+                  }
+                },
+                "contentVersion": "1.0.0.0",
+                "outputs": {}
```
[the remark this answers is not in the document]
Comment (Abuen) [reply solutions/remotemonitoring/armtemplates/standard.json]: This provides the Jason schema that reads the HTTP post request and allows the Dynamic Content feature of the logic app to pass that info onto the send_an_email connector

File: solutions/remotemonitoring/armtemplates/standard.json [hunk 3]
```diff
@@ -855,7 +868,98 @@
...
+              },
+              "parameters": {
+                "$connections": {
+                  "value": {
+                    "office365": {
+                      "id": "[concat(subscription().id, '/providers/Microsoft.Web/locations/', 'eastus', '/managedApis/', 'office365')]",
```
[the remark this answers is not in the document]
Comment (Abuen) [reply solutions/remotemonitoring/armtemplates/standard.json]: Will change that and push

File: solutions/remotemonitoring/armtemplates/standard.json [hunk 4]
```diff
@@ -335,7 +335,20 @@
                 "description": "Bing Maps region",
                 "comments": "Marked for deletion, unused"
```
Comment (ppathan) [inline solutions/remotemonitoring/armtemplates/standard.json]: you also need to update basic.json

Comment (Abuen) [reply solutions/remotemonitoring/armtemplates/standard.json]: added the changes

**Pull request closed without merging**